src/csim/update_ops_control_single_target_single.cpp

```diff
@@ -251,3 +243,111 @@ void single_qubit_control_single_qubit_dense_matrix_gate_simd(
     }
 }
 #endif
+
+#ifdef _USE_SVE
+static inline void MatrixVectorProduct2x2(svbool_t pg, svfloat64_t in00r,
+    svfloat64_t in00i, svfloat64_t in11r, svfloat64_t in11i, svfloat64_t mat02r,
+    svfloat64_t mat02i, svfloat64_t mat13r, svfloat64_t mat13i,
+    svfloat64_t* out01r, svfloat64_t* out01i);
```

Comment:
Thanks for the PR. I am still in the process of reading it, but I found some unnecessary code, so I will comment on it.
I think this prototype declaration is unnecessary because the function is defined below.

```suggestion
```

Reply:
Thank you for pointing this out. Indeed, this prototype declaration is unnecessary. I merge the modifications you suggested.